File: Answer/1980.找出不同的二进制字符串.java
/*
 * @lc app=leetcode.cn id=1980 lang=java
 *
 * [1980] 找出不同的二进制字符串
 */

// @lc code=start
class Solution {

    public String findDifferentBinaryString(String[] nums) {
        var n = nums.length;
        var res = new char[n];
        for (var i = 0; i < n; i++) {
            res[i] = nums[i].charAt(i) == '0' ? '1' : '0';
        }
        return new String(res);
    }
}
// @lc code=end

File: Answer/1980.找出不同的二进制字符串.go
/*
 * @lc app=leetcode.cn id=1980 lang=golang
 *
 * [1980] 找出不同的二进制字符串
 */

// @lc code=start
func findDifferentBinaryString(nums []string) string {
	n := len(nums)
	res := make([]byte, n)
	for i := 0; i < n; i++ {
		if nums[i][i] == '0' {
			res[i] = '1'
		} else {
			res[i] = '0'
		}
	}
	return string(res)
}

// @lc code=end
